packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -47,9 +47,6 @@ const config: HardhatUserConfig = {
   },
   foundry: {
     buildInfo: true,
-    // call runSuper in the hardhat task so that
-    // other things execute
-    runSuper: true,
```

Comment:
What's the effect of removing this?

Reply:
It speeds up compilation speed by not calling `runSuper` in the hh task. Since we no longer are using hh-typechain, we don't need to call `runSuper`. Its about 50% faster with this removed